packages/design-system/src/components/Dialog/Dialog.stories.jsx

```diff
@@ -40,21 +41,20 @@ export default {
   },
 };
 
-export const DialogExample = ({ ...args }) => {
-  const [shown, setShown] = useState();
-  const showModal = () => setShown(true);
-  const hideModal = () => setShown(false);
+export const DialogExample = ({ isOpen: _isOpen, ...args }) => {
+  const [{ dialogOpen }, updateArgs] = useArgs();
```

Comment:
I got this warning in the console:
```
React does not recognize the `dialogOpen` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `dialogopen` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
```

Reply:
Thanks for bringing that to my attention. Getting this args stuff to work right has been a challenge. I've got the fix, though, and I'm going to push it up after my local visual regression tests finish updating.